pandas/tests/frame/test_subclass.py

```diff
@@ -557,3 +557,26 @@ def strech(row):
         result = df.apply(lambda x: [1, 2, 3], axis=1)
         assert not isinstance(result, tm.SubclassedDataFrame)
         tm.assert_series_equal(result, expected)
+
+    def test_subclassed_df_reduce_functions(self):
```

Comment:
can this test be parametrized and split to use all_numeric_reductions and all_boolean_reductions fixtures from conftest.py

Reply:
Thanks for the feedback. This is done with the most recent commit. Waiting for the automated tests to mark this as resolved.